### DISCUSSION TITLE
[Enhancement] [date-picker] [Feature Requests] DateTimePicker Support footer slot.

### DISCUSSION THREAD

**Discussion by wuhaohuang** [element-plus/element-plus#16219] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

Currently, the lower part of the component (as shown in the picture) does not support user customization. We hope to support user-defined UI through slots.

<img width="521" alt="image" src="https://github.com/element-plus/element-plus/assets/19322584/f2afc70c-9001-4d31-b24f-3a6728aa212a">

**Comment by Aseisman** [comment]
![image](https://github.com/element-plus/element-plus/assets/38846995/faeff05d-7078-4d25-b2ea-73ce0c5921cd)
I think,,,, in this way can slove our problems

**Reply by Aseisman** [reply]
waiting for the dalaos !~~~

**Comment by Aseisman** [comment]
look here  https://github.com/element-plus/element-plus/pull/16253

**Reply by wuhaohuang** [reply]
wow, but it seems like not ready yet

**Reply by Aseisman** [reply]
how to make pull-requests pass faster

**Reply by wuhaohuang** [reply]
i don' know. judging from the content, may be that some pre-checks have failed, so you can consult the official.

**Reply by wuhaohuang** [reply]
早上看了下你的提交，可能不太符合预期。所以我自己写了下提了 PR 了。
#16378
